core/src/utils/forms.ts

```diff
@@ -320,14 +320,20 @@ const buildXsd = (
 
     if (property) {
       const isArray = property.type === 'array'
-      const type = isArray && property.items ? property.items.type : property.type
+      const hasMultipleTypes =
```

Comment:
@radeno  so, the following is true with this code, right ? Or am I missing something else:

```suggestion
      // a limited support for JSON Schema multi types - https://json-schema.org/draft/2020-12/json-schema-validation#section-6.1.1 - allowing for "nullable" types. `null` type must be the latter one in the type array pair.
      const hasMultipleTypes =
```

if so, I'd add the comment and otherwise don't mind merging it as is. 

In fact, would move it forward asap, as the plan for the following week is to simplify the build process of the lib, which will involve merging the core & cli parts into single package (we're no longer using the core separately) - that would mean anything kept open would likely cause conflicts.

Reply:
Agree. let's move forward. I like your comment that multiple types are limited only to support only nullable fields.

Reply:
My future PR should be about support JSON schema attributes `minContains` and `maxContains` and complex array of objects than just primitive types. This unlocks complex repeatable fields.